src/styles/card/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+  }
+
+  &-container {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+    align-items: center;
+  }
+
+  &-container__reviews {
+    font-weight: 400;
+    font-size: 10px;
+    margin: 2px 0 0;
+    color: #060b35;
+  }
+
+  &--0 {
+    .stars__star:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg); 
```

Comment:
You are changing the background image of the star to an active state, but the selector seems to be incorrect. The `nth-child(-n + 4)` selector will select the first 4 children, but since this is a modifier for 0 active stars, it should not select any. Consider fixing the selector to match the correct number of active stars for each modifier.